linux/lnxcontroller.cpp

```diff
@@ -708,8 +711,7 @@ gameController *CreateController(int num_funcs, ct_function *funcs, char *remote
 }
 
 void DestroyController(gameController *ctl) {
-  if (ctl)
```

Comment:
why do we remove null check?

Reply:
`delete []` operator correctly handles `nullptr`, checking before deletion is useless.